pkg/threescale/backend/backend_test.go

```diff
@@ -218,6 +219,156 @@ func TestBackend_Authorize(t *testing.T) {
 			},
 			expectResult: &threescale.AuthorizeResult{Authorized: false},
 		},
+		{
+			name: "Test the application of policy, fail closed case - timeout error",
+			setup: func(cacheable Cacheable, remoteClient *mockRemoteClient) *Backend {
+				remoteClient.err = &net.DNSError{
```

Comment:
Ok, I assume no other errors are worth trying, since most that I can see are just the `error` interface with just a textual description.

Reply:
Yes correct, I just picked the struct with exported fields to making testing simple, but we will be doing a type assertion against the `net` error at runtime so it will come from there.